### PR TITLE
sync nordstrom en language

### PR DESCRIPTION
**Pull request by dapengJacky**
Signed-off-by: jacky <peng.chen@daocloud.io>
sync nordstorm en language

### CODE REVIEW

**Comment by Sea-n** [comment]
Please don't change English version in this PR.

**Comment by dapengJacky** [comment]
> Please don't change English version in this PR.

OK

File: content/zh-cn/case-studies/nordstrom/index.html [hunk 1]
```diff
@@ -2,151 +2,156 @@
...
+-->
+<p>为 Nordstrom 构建 Kubernetes 企业平台的团队高级工程师 Dhawal Patel 说，“使用 Kubernetes 的 Nordstrom 技术开发人员现在项目部署得更快，并且能够只专注于编写应用程序。”此外，该团队还提高了运营效率,根据工作负载将 CPU 利用率从 5 倍提高到 12 倍。Patel 说：“我们运行了数千台虚拟机 (VM),但无法有效地使用所有这些资源。借助 Kubernetes,我们甚至不需要尝试去提高集群的效率,就能使运营效率增长 10 倍。”</p>
+
+<!-- 
+{{< case-studies/quote author="Dhawal Patel, senior engineer at Nordstrom" >}}
+"We are always looking for ways to optimize and provide more value through technology. With Kubernetes we are showcasing two types of efficiency that we can bring: Dev efficiency and Ops efficiency. It's a win-win."
+{{< /case-studies/quote >}}
+-->
+{{< case-studies/quote author="Dhawal Patel, Nordstrom 高级工程师" >}}
 “我们一直在寻找通过技术进行优化和提供更多价值的方法。通过 Kubernetes ，我们在开发效率和运营效率这两方面取得了示范性的提升。这是一个双赢。”
```
Comment (Sea-n) [inline content/zh-cn/case-studies/nordstrom/index.html:52]:
```suggestion
“我们一直在寻找通过技术进行优化和提供更多价值的方法。通过 Kubernetes，我们在开发效率和运营效率这两方面取得了示范性的提升。这是一个双赢。”
```

File: content/zh-cn/case-studies/nordstrom/index.html [hunk 2]
```diff
@@ -2,151 +2,156 @@
...
-</div>
-</section>
-<div class="banner3" style="background-image: url('/images/case-studies/nordstrom/banner3.jpg')">
-<div class="banner3text">
-<!-- "We made a bet that Kubernetes was going to take off, informed by early indicators of community support and project velocity, so we rebuilt our system with Kubernetes at the core," -->
+{{< /case-studies/quote >}}
+
+<!-- 
+<p>When Dhawal Patel joined <a href="http://shop.nordstrom.com/">Nordstrom</a> five years ago as an application developer for the retailer's website, he realized there was an opportunity to help speed up development cycles.</p> -->
+<p>当 Dhawal Patel 五年前加入<a href="http://shop.nordstrom.com/"> Nordstrom </a>，担任该零售商网站的应用程序开发人员时，他意识到有机会帮助加快开发周期。</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/nordstrom/index.html:57]:
```suggestion
<p>当 Dhawal Patel 五年前加入 <a href="http://shop.nordstrom.com/">Nordstrom</a>，担任该零售商网站的应用程序开发人员时，他意识到有机会帮助加快开发周期。</p>
```

File: content/zh-cn/case-studies/nordstrom/index.html [hunk 3]
```diff
@@ -2,151 +2,156 @@
...
+
+<!-- 
+<p>In those early DevOps days, Nordstrom Technology still followed a traditional model of silo teams and functions. "As a developer, I was spending more time fixing environments than writing code and adding value to business," Patel says. "I was passionate about that—so I was given the opportunity to help fix it."</p>
+-->
+<p>在早期的 DevOps 时代,，Nordstrom 技术仍然遵循传统的孤岛团队和功能模型。Patel 说：“作为开发人员，我花在维护环境上的时间比编写代码和为业务增加价值的时间要多。我对此充满热情，因此我有机会参与帮助修复它。”</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/nordstrom/index.html:62]:
```suggestion
<p>在早期的 DevOps 时代，Nordstrom 技术仍然遵循传统的孤岛团队和功能模型。Patel 说：“作为开发人员，我花在维护环境上的时间比编写代码和为业务增加价值的时间要多。我对此充满热情，因此我有机会参与帮助修复它。”</p>
```

File: content/zh-cn/case-studies/nordstrom/index.html [hunk 4]
```diff
@@ -2,151 +2,156 @@
...
+{{< /case-studies/quote >}}
+-->
+{{< case-studies/quote >}}
+“借助 Kubernetes ，我们甚至不需要尝试去提高集群的效率，目前 CPU 利用率为 40%，较之前增长了 10 倍。我们正在运行 2600 多个客户 Pod，如果它们直接进入云，这些 Pod 将是 2600 多个 VM。我们现在在 40 台 VM 上运行它们，因此这大大降低了运营开销。
+{{< /case-studies/quote >}}
+
+<!-- 
+<p>Speed is great, and easily demonstrated, but perhaps the bigger impact lies in the operational efficiency. "We run thousands of VMs on AWS, and their overall average CPU utilization is about four percent," says Patel. "With Kubernetes, without even trying to make our cluster efficient, we are currently at 40 percent CPU utilization—a 10x increase. We are running 2600+ customer pods that would have been 2600+ VMs if they had gone directly to the cloud. We are running them on 40 VMs now, so that's a huge reduction in operational overhead."</p>
+-->
+<p>速度很重要，并且很容易证明，但也许更大的影响在于运营效率。Patel 说：“我们在 AWS 上运行了数千个 VM ，它们的总体平均 CPU 利用率约为 4%。借助 Kubernetes ，我们甚至不需要尝试去提高集群的效率，目前 CPU 利用率为 40%，较之前增长了 10 倍。我们正在运行 2600 多个客户 pod ，如果它们直接上云，这些 Pod 将是 2600 多个 VM。我们现在在 40 台 VM 上运行它们，因此这大大降低了运营开销。</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/nordstrom/index.html:142]:
```suggestion
<p>速度很重要，并且很容易证明，但也许更大的影响在于运营效率。Patel 说：“我们在 AWS 上运行了数千个 VM，它们的总体平均 CPU 利用率约为 4%。借助 Kubernetes，我们甚至不需要尝试去提高集群的效率，目前 CPU 利用率为 40%，较之前增长了 10 倍。我们正在运行 2600 多个客户 pod ，如果它们直接上云，这些 Pod 将是 2600 多个 VM。我们现在在 40 台 VM 上运行它们，因此这大大降低了运营开销。</p>
```

**Review by Sea-n — COMMENTED** [review]
Please check the punctuation and spaces, thanks.

**Comment by dapengJacky** [comment]
> Please check the punctuation and spaces, thanks.

OK,thanks for your check,I have updated punctuation,spaces and parentheses;Please review again,thanks

**Pull request merged**